spec/lib/deployers_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 RSpec.describe Deployers do
   around do |ex|
-    ClimateControl.modify DEPLOYERS: 'one,two,three,four,five,six,seven' do
+    ClimateControl.modify DEPLOYERS: '[{"name":"One","userId":"1"},{"name":"Two","userId":"2"},{"name":"Three","userId":"3"},{"name":"Four","userId":"4"},{"name":"Five","userId":"5"}]' do
```

Comment:
What are `name` and `userId`? From Slack? How do we get them?

Reply:
name: is just a display name,
userId: is the slack userId of a team member, which I have configured in the the Azure DevOps library group
You can view the slack userId of a person by navigating to their profile from inside slack.

Reply:
Ah ok. Thanks. Could we rename them something more explicit like `nameOnDashboard` and `slackUserId`?

Reply:
@duncanjbrown renamed